example/src/layouts/MainLayout.vue

```diff
@@ -22,6 +26,13 @@ const currentPath = inject("currentPath")
         </div>
         <VLink class="btn btn-secondary ms-auto" to="/">New Plan</VLink>
         <VLink class="btn btn-link" to="/about">About</VLink>
+        <button
+          class="btn btn-link theme-toggle"
+          @click="toggleTheme"
+          :title="theme === 'light' ? 'Switch to dark mode' : 'Switch to light mode'"
+        >
+          <FontAwesomeIcon :icon="theme === 'light' ? faMoon : faSun" />
```

Comment:
In my opinion, this would be better the other way around. ie. the icon should show the current state.